go.mod

```diff
@@ -11,4 +11,5 @@ require (
 	k8s.io/client-go v0.18.2
 	k8s.io/component-base v0.18.2
 	k8s.io/klog v1.0.0
+	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6 // indirect
```

Comment:
I don't think we need this.

Reply:
This change comes from running `make gen` (the generator binaries have updated). Since we verify the vendor directory, it has to be committed to pass CI.

We discussed pinning these generators to a specific version before, so that these random vendor/go.mod updates don't happen, but haven't made any progress on that yet.